stylesheets/scss/layouts/_gui.scss

```diff
@@ -145,7 +156,32 @@
   }
 
   .drawflow .connection .main-path {
-    width: 2px;
+    width: 3px;
+  }
+
+  .drawflow .connection {
+
+    // @PF: Colors for different connection types
```

Comment:
@bmlancien - these comments you can also make them on github (hover over the left of the line and click on the litle plus square :)